give.lua

```diff
@@ -281,11 +281,8 @@ local function GiveItemCommand( Split, Player, SafeCommand )
 
 			-- Set Lore if given
 			if DataTagTable.display.Lore then
-				local Lore = ""
-				for _, value in ipairs(DataTagTable.display.Lore) do
-					Lore = Lore .. value .. "`"  -- Newline is '`' character rather than "\n"
-				end
-				Item.m_Lore = Lore
+				local Lore = DataTagTable.display.Lore
```

Comment:
What is the point of this Lore variable? Why not directly assign it as:

    Item.m_LoreTable = DataTagTable.display.Lore